ansible/roles/accumulo/templates/accumulo-cluster-systemd

```diff
@@ -287,26 +292,35 @@ function stop_all() {
   done
   wait
 
+{% if accumulo_version is version('2.1.0','<') %}
   for tracer in $(grep -E -v '(^#|^\s*$)' "${conf}/tracers"); do
     stop_service "$tracer" tracer &
   done
   wait
+{% endif %}
 
   # stop tserver still running
   stop_tservers
 
   echo "Cleaning all server entries in ZooKeeper"
+{% if accumulo_version is version('2.1.0','<') %}
   ${accumulo_cmd} org.apache.accumulo.server.util.ZooZap -master -tservers -tracers
+{% else %}
+  ${accumulo_cmd} org.apache.accumulo.server.util.ZooZap -manager -tservers
+{% endif %}
```

Comment:
Wondering if this would be better as a single line or not?
```suggestion
  ${accumulo_cmd} org.apache.accumulo.server.util.ZooZap -master -tservers{{ ' -tracers' if accumulo_version is version('2.1.0','<') }}
```

Reply:
I had also thrown in that manager change, but that does not have to be done in this PR

Reply:
Oops, yeah I glossed over the master/manager change too. Makes more sense the way you have it now.